### ISSUE TITLE
Trouble getting Mplus to run on Mac and ST 3

### ISSUE THREAD

**Issue by psychstatistics** [bkeller2/Mplus#10]
I've successfully installed the package. When I open an .inp file I get the syntax highlighting. I am not having any luck getting Mplus to run using the Build command. Basically nothing happens.

I am on Yosemite and using Mplus 7. Do you have any ideas or suggestions? I can provide more information if that would be helpful.

Thanks.
Scott

**Comment by bkeller2** [comment]
Hello Scott,

What directory is your Mplus installed into?

If it is not in '/Applications/Mplus/mplus' then it will not run out of the box.

This is not a big issue to change, specifically go to the file:

Mplus/Setting Files/Mplus.sublime-build

And edit line 8:
		"cmd": ["/Applications/Mplus/mplus \"$file\" ; subl \"${file/\\.inp/\\.out/}\""]

To have the appropriate 'mplus' executable (note this should be called just mplus, not the editor itself).

Furthermore, the same needs to be done for sublime text, which is also defaulted in the application folder. Thus you will also need to edit line 7 if sublime text is not in that folder:

		"path": "/Applications/Sublime Text 2.app/Contents/SharedSupport/bin/:/Applications/Sublime Text.app/Contents/SharedSupport/bin/:$PATH",

The part you need to change here is just the "/Applications/Sublime Text.app/" portion to point towards the appropriate sublime text app bundle (with .app suffix).

This is my first guess as to why nothing is working.
If this is not the case I can investigate further, but currently it is working for me on Yosemite with both ST2 and ST3.

Thank you and best,
Brian

**Comment by psychstatistics** [comment]
Thanks for your help. I made those changes and still couldn't get it working. I removed the package and reinstalled manually per your README file and now things work. I'm wondering if I have something wrong with my Sublime Install that is causing problems with the package installer. In any case, I will dig into the issue a little more and if there is something that seems like could be associated with your specific package (rather than a problem with my install), I will let you know.

Thanks for your work on this. Awesome package!

Scott
